src/olympia/scanners/migrations/0004_auto_20191018_0740.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+from django.db.models import Case, Value, When
+
+from olympia.constants.scanners import YARA
+
+
+def move_yara_results(apps, schema_editor):
+    YaraResult = apps.get_model('yara', 'YaraResult')
+    ScannerResult = apps.get_model('scanners', 'ScannerResult')
+
+    for yara_result in YaraResult.objects.all():
```

Comment:
I wonder if we could do it in one SQL query, either with raw SQL or with fancy ORM tricks. Otherwise this is going to take a while...

Reply:
Yeah, that's true.

FWIW, there are 10k entries in production.

Reply:
I switched to a SQL query to copy the data.